buildspec/linuxIntegrationTests.yml

```diff
@@ -13,41 +13,19 @@ env:
 
 phases:
   install:
-    runtime-versions:
-      java: corretto11
-      dotnet: 3.1
-
     commands:
-      # Added for Python2 for Ubuntu 20.4
-      - add-apt-repository universe
-      # WORKAROUND for yarn key being out of date, update keys so we can run builds TODO remove dirmngr and apt-key commands
-      - apt-get install dirmngr
-      - apt-key adv --refresh-keys --keyserver keyserver.ubuntu.com
-      - apt-get update
-      - apt-get install -y jq python2.7 python3.6 python3.7 python3.8 python3-pip python3-distutils
-      # As of Ubuntu 20.4, there is no longer pip for Python 2, so we need to install it manually
-      - curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py
-      - python2 get-pip.py
+      - startDocker.sh
       - aws sts assume-role --role-arn $ASSUME_ROLE_ARN --role-session-name integ-test > creds.json
       - export KEY_ID=`jq -r '.Credentials.AccessKeyId' creds.json`
       - export SECRET=`jq -r '.Credentials.SecretAccessKey' creds.json`
       - export TOKEN=`jq -r '.Credentials.SessionToken' creds.json`
-      - pip3 install --user --upgrade  aws-sam-cli
```

Comment:
worth making this part work later?

Reply:
Upgrading? I guess we should so we can test new runtimes and that jazz and not wait for the docker image to update...

Should probably move the logic into a shell script so its not duplicated.